content/logs/docker.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Docker log collection
+kind: Documentation
+further_reading:
+- link: "/logs/explore"
+  tag: "Documentation"
+  text: Learn how to explore your logs
+- link: "/logs/processing"
+  tag: "Documentation"
+  text: Learn how to process your logs
+- link: "/logs/parsing"
+  tag: "Documentation"
+  text: Learn more about parsing
+---
+
+<div class="alert alert-info">
+Datadog's Logs is currently available via public beta. You can apply for inclusion in the beta via <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+
+## Overview
+
+The Datadog Agent can collect logs from containers starting at the version 6. Two installations are possible:
```

Comment:
Datadog Agent version 6 and greater can collect logs from containers.

Reply:
👍